app/src/org/commcare/activities/components/ImageCaptureProcessing.java

```diff
@@ -36,6 +36,7 @@ public class ImageCaptureProcessing {
      * @return A pair containing raw image and scaled imagePath. The first entry is the raw image
      * while the second one is path to scaled image.
      */
+    @SuppressLint("ExifInterface")
```

Comment:
Sorry for the confusion but we don't want any `SuppressLint` annotations in the code even if the lint check breaks as we still want to know about all the lint errors in the code but just want to fix the ones that's been introduced by the new code in PR.